Packages/vcs/vcs/Canvas.py

```diff
@@ -4961,6 +4979,15 @@ def setcolormap(self, name):
         #   updateVCSsegments_flag = args[1]
         # except:
         #   updateVCSsegments_flag = 1
+        # Python < 3 DeprecationWarning ignored by default
```

Comment:
@aashish24 I don't think this should be here. We should issue this warning whenever the user sets the colormap instead.